db/schema.rb

```diff
@@ -102,12 +102,12 @@
     t.boolean  "is_calibrated",                            default: false
     t.boolean  "is_selfreview_enabled"
     t.string   "reputation_algorithm",       limit: 255,   default: "Lauw"
+    t.integer  "simicheck",                  limit: 4,     default: -1
```

Comment:
Are these changes to `schema.rb` necessary? It doesn't appear to be directly related to the other changes you've made.